dataland-frontend/src/assets/scss/components/datatable.scss

```diff
@@ -2,10 +2,15 @@
   border-radius: 0;
   background: $table-background-color;
   color: $main-text-color;
+  .onlyHeaders {
+    .p-datatable-tbody {
+      display: none;
+    }
+  }
   tr {
-    border-style: solid;
-    border-width: 0 0 1px 0;
-    border-color: $table-border-dark;
+    &:not(.p-rowgroup-header) {
```

Comment:
Border seems like the more natural choice. However, why wasn't it visible before at e.g. the /copanies page?
If you are anyways on it. Why not add the broader underline on the table headers at the /companies page.

Reply:
Doesn't 1px imply some blur? And 0px none?

Reply:
Not sure, i would like to hand this over to @MarcinStanczyk7 who has done the changes and knows these code changes better than me.

Reply:
I removed that 1px blur.

Reply:
What about the other questions?

Reply:
I was just curious. Did you notice a different appearance?

Reply:
In zooming, yes.